demisto_sdk/commands/lint/linter.py

```diff
@@ -1389,6 +1393,7 @@ def _check_native_image_flag(self, docker_image_flag):
             DockerImageFlagOption.NATIVE_DEV.value,
             DockerImageFlagOption.NATIVE_GA.value,
             DockerImageFlagOption.NATIVE_MAINTENANCE.value,
+            DockerImageFlagOption.NATIVE_TARGET.value,
```

Comment:
I you added this value here, I think we need to add it to the error message and to the function's docstring as a possible option.